kalliope/core/OrderAnalyser.py

```diff
@@ -74,6 +77,23 @@ def get_matching_synapse(cls, order, brain=None):
                         except KeyError:
                             logger.debug("[OrderAnalyser] Warning, missing parameter 'matching-type' in order. "
                                          "Fallback to 'normal'")
+                        try:
+                            stt_correction = signal.parameters["stt-correction"]
+                            logger.debug("[OrderAnalyser] stt-correction provided by user")
+                        except KeyError:
+                            logger.debug("[OrderAnalyser] No stt-correction provided")
+
+                        try:
+                            stt_correction_file = signal.parameters["stt-correction-file"]
+                            logger.debug("[OrderAnalyser] stt-correction-file provided by user")
+                        except KeyError:
+                            logger.debug("[OrderAnalyser] No stt-correction-file provided")
+
+                    if stt_correction_file is not None:
+                        stt_correction = cls.load_stt_correction_file(stt_correction_file)
+
+                    if stt_correction is not None:
```

Comment:
As far as I understand, in case we have stt_correction and stt_correction_file, the file overrides the stt_correction. I would rather go the opposite way; having the stt_correction over the file. 
It allows the user to define a global file to reused in different places, and override with stt_correction in specific cases.

Reply:
You are totally right!
I'll update this tonight.